tests/test_pipenv.py

```diff
@@ -651,3 +652,26 @@ def test_urls_work(self):
             dep = p.lockfile['default'][key]
 
             assert 'file' in dep
+
+    @pytest.mark.install
+    @pytest.mark.files
+    def test_local_zipfiles(self):
+
+        with PipenvInstance() as p:
+            zip_contents = requests.get('https://github.com/divio/django-cms/archive/release/3.4.x.zip')
```

Comment:
I'd prefer to either find a way to stub this out or bundle a condensed example.zip package if we can. Network calls tend to be where a lot of test time and failures come from.

Reply:
Sure I wasn't totally sure about the best practice for this so I just copied the test above it (though to be fair that is testing a URL as well).  I didn't want to just dump this in the repo without talking ot someone though.  Might want to find something smaller to be honest.  Where should I put it?

Reply:
Sorry I missed this last night @techalchemy. I'd create a directory under tests with "test_artifacts" are something like that and drop it in there.